### PR TITLE
APPSERV-22 Downloads JavaDB and necessary SQL script and add it to the Payara VI folder

### PR DESCRIPTION
**Pull request by MeroRai**

### CODE REVIEW

File: patch/run_jakartaeetck.sh [hunk 1]
```diff
@@ -236,6 +236,16 @@ if [ ! -d "${CTS_HOME}/vi/$GF_VI_TOPLEVEL_DIR" ]; then
   exit 1
 fi
 
+wget --progress=bar:force --no-cache http://192.168.1.120:8080/userContent/tck/javadb.zip -O ${CTS_HOME}/javadb.zip
```
Comment (pdudits) [inline patch/run_jakartaeetck.sh]: Can you please update it in a way that it also works in standalone environment:

* The download location is specified with `DERBY_URL`
* The default is specified in `functions.sh/init_urls()` (would be `$BASE_URL/javadb.zip`
* Update `bundles/run_server.sh` to warn when javadb.zip is not present in bundles directory.

File: functions.sh [hunk 2]
```diff
@@ -57,7 +57,16 @@ init_urls () {
     fi
     if [ -z "$DI_TCK_URL" ]; then
         DI_TCK_URL=$BASE_URL/jakarta.inject-tck-1.0-bin.zip
-    fi    
+    fi
+    if [ -z "$DERBY_URL" ]; then
+        DERBY_URL=$BASE_URL/javadb.zip
+    fi
+    if [ -z "$EJBTIMER_DERBY_SQL" ]; then
+        EJBTIMER_DERBY_SQL=$BASE_URL/ejbtimer_derby.sql.zip
```
Comment (pdudits) [inline functions.sh]: Without the .zip, I suppose

File: patch/run_jakartaeetck.sh [hunk 3]
```diff
@@ -237,6 +241,16 @@ if [ ! -d "${CTS_HOME}/vi/$GF_VI_TOPLEVEL_DIR" ]; then
   exit 1
 fi
 
+wget --progress=bar:force --no-cache $DERBY_URL -O ${CTS_HOME}/javadb.zip
```
Comment (pdudits) [inline patch/run_jakartaeetck.sh]: When basing this on current master, at this section into `/jakartaeetck.sh` at similar location

File: run.sh [hunk 4]
```diff
@@ -112,7 +112,7 @@ if [ -z "$SKIP_TEST" ]; then
   echo "Environment"
   printenv
   echo "Starting test!"
-  time bash -x $SCRIPTPATH/jakartaeetck.sh "$@" |& tee $CTS_HOME/$TEST_SUITE.log
+  time bash -x patch/run_jakartaeetck.sh "$@" |& tee $CTS_HOME/$TEST_SUITE.log
```
Comment (pdudits) [inline run.sh]: Do not apply this to master, we already have our own version of runner

File: bundles/run_server.sh [hunk 5]
```diff
@@ -11,4 +11,7 @@ check_present latest-glassfish.zip
 check_present payara-prerelease.zip
 check_present cdi-tck-2.0.6-dist.zip
 check_present jakarta.inject-tck-1.0-bin.zip
+check_present javadb.zip
+check_present ejbtimer_derby.sql
+check_present jsr352-derby.sql
```
Comment (pdudits) [inline bundles/run_server.sh]: Please include the files in the PR  as well (git might fight with you a bit, as .zip is in .gitignore), so standalone user, like me debugging failures, don't need to go hunting for them.

File: run.sh (hunk 4 above)
Comment (pdudits) [reply run.sh]: Ah you already merged master in. Then definetely change the script in root.

File: bundles/run_server.sh (hunk 5 above)
Comment (MeroRai) [reply bundles/run_server.sh]: I will add them to the `bundles` folder.

**Comment by pdudits** [comment]
I'll set up openjdk on the build machines later tonight and run it

**Pull request merged**